src/styles/index.scss

```diff
@@ -1,3 +1,215 @@
+:root {
+  --accent-color: #00acdc;
+  --main-text-color: #616070;
+  --accent-text-color: #060b35;
+  --white: #fff;
+  --border-color: #f3f3f3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+/* #region header */
+
+ul {
+  list-style: none;
+}
+
+.header {
+  box-sizing: content-box;
+  display: flex;
+  justify-content: space-between;
+  height: 40px;
+  align-items: center;
+  padding: 10px 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
```

Comment:
According to the code style guide, the block of a selector should be separated from the previous block by a newline. Please add a newline before the `.header` selector for consistency and readability.